docs/source/features/version_control/draft.rst

```diff
@@ -23,18 +23,24 @@ in the SDK. Of course, it is necessary to get a dataset first:
 *****************
 
 SDK provides method :meth:`~graviti.manager.draft.DraftManager.create` to support creating a
-draft based on a branch. The title, description and branch of the draft can be specified, but
-only title is required:
+draft based on a branch:
 
 .. code:: python
 
-   dataset.drafts.create(f"{DRAFT_TITLE}")
-   dataset.drafts.create(f"{DRAFT_TITLE}", f"{DRAFT_DESCRIPTION}")
    dataset.drafts.create(f"{DRAFT_TITLE}", branch=f"{BRANCH_NAME}")
 
-.. note::
-   If no branch is specified, the created draft will be based on the current branch of the
-   dataset, which can be viewed by ``dataset.HEAD``.
+If no branch is specified, the created draft will be based on the current branch of the
+dataset, which can be viewed by ``dataset.HEAD``:
```

Comment:
Do we need to add a link to the API reference of `HEAD`?

Reply:
The description about HEAD will be modified together in another pr